server/graphql/resolvers.ts

```diff
@@ -151,24 +151,32 @@ const resolvers = {
     },
     createClient: (_, { client }, { dataSources }): Promise<ServerResponseInterface> => dataSources.clients.create(client),
     updateClient: (_, { client }, { dataSources }): Promise<ServerResponseInterface> => dataSources.clients.update(client),
-    softDeleteClient: (_, { id }, { dataSources}): Promise<ServerResponseInterface> => dataSources.clients.softDelete(id)
+    softDeleteClient: (_, { id }, { dataSources }): Promise<ServerResponseInterface> => dataSources.clients.softDelete(id)
   },
   RequestGroup: {
-    numOpen: (parent, __, { dataSources }): Number => 
-      parent.requestTypes.map(id => {
+    numOpen: (parent, __, { dataSources }): Number => {
+      const openRequestsInRequestTypes = parent.requestTypes.map(id => {
         return filterOpenRequests(getRequestsById(dataSources.requestTypes.getById(Types.ObjectId(id)).requests, dataSources)).length
-      }).reduce((total, num) => total + num, 0),
-    hasAnyRequests: (parent, __, { dataSources }): Boolean => 
-      parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)).requests.length).reduce((notEmpty, len) => notEmpty || len > 0),
```

Comment:
The issue here was simply that reduce requires an initial value for empty input. Simply replace

".reduce((notEmpty, len) => notEmpty || len > 0),"

with 

".reduce((notEmpty, len) => notEmpty || len > 0, false),"

to avoid overcomplicating the function.

Reply:
icic, i'll fix this